src/DynamoCoreWpf/ViewModels/Core/ConnectorPinViewModel.cs

```diff
@@ -138,39 +138,42 @@ public bool IsSelected
             get { return model.IsSelected; }
         }
 
-        private bool isVisible;
+        private bool isCollapsed;
         [JsonIgnore]
-        public bool IsVisible
+        public override bool IsCollapsed
         {
-            get
-            {
-                return isVisible;
-            }
+            get => isCollapsed;
             set
             {
-                isVisible = value;
-                RaisePropertyChanged(nameof(IsVisible));
-                RaisePropertyChanged(nameof(IsPartlyVisible));
+                if (isCollapsed == value)
+                {
+                    return;
+                }
+
+                isCollapsed = value;
+                RaisePropertyChanged(nameof(IsCollapsed));
+                RaisePropertyChanged(nameof(IsTemporarilyVisible));
```

Comment:
Can you remind me why we need to raise change here if the value is not touched?

Reply:
Fixed- this was a leftover.